README.md

```diff
@@ -46,7 +46,7 @@ make iso
 * Docker
 
 ```
-make docker:iso
+make docker:boot.iso
```

Comment:
We also have to update the above `make iso` command.

Reply:
ok